app/client/ui/UserManager.ts

```diff
@@ -518,90 +525,57 @@ export class UserManager extends Disposable {
   }
 }
 
+
 /**
  * Represents the widget that allows typing in an email and adding it.
  * The border of the input turns green when the email is considered valid.
  */
```

Comment:
We can move this down a bit so that it's right above `ACMemberEmail`.
```suggestion
/**
 * Represents the widget that allows typing in an email and adding it.
 */
```